network/src/connectivity_manager/mod.rs

```diff
@@ -259,8 +270,8 @@ where
         //    connection with a peer.
         let mut pending_dials = FuturesUnordered::new();
 
-        // When we first startup, let's attempt to connect with our seed peers.
-        self.check_connectivity(&mut pending_dials).await;
+        let ticker = self.time_service.interval(self.connectivity_check_interval);
+        tokio::pin!(ticker);
```

Comment:
So, now just wait for the first interval to connect to seed peers?  Never really understood why we did this manually here except I guess "slightly faster"

Reply:
it's actually the same, since the first interval tick triggers immediately (t=0)